client/asset/dcr/dcr.go

```diff
@@ -5134,6 +5174,125 @@ func (dcr *ExchangeWallet) EstimateSendTxFee(address string, sendAmount, feeRate
 	return finalFee, isValidAddress, nil
 }
 
+func (dcr *ExchangeWallet) isSPV() bool {
+	return dcr.walletType == walletTypeSPV
```

Comment:
This reminds me. There's a `dcr.wallet.SpvMode()` which may be true even for a wallet connected via rpc. Put another way, isSPV could be true even if walletType isn't walletTypeSPV.